### PR TITLE
Pathing

### PR DESCRIPTION
**Pull request by jonathanjtan**
i hate creep block

### CODE REVIEW

File: poll.py [hunk 1]
```diff
@@ -26,14 +26,13 @@ def lambda_handler(event, context):
...
-            if not Path:
-                Path = 'NULL'
             Filename = body['fileName']
             RequestedFormats = body['sizeFormat']
-            if Path is not 'NULL':
-                VideoURL = "https://{}.s3.amazonaws.com/{}{}".format(Bucket, Path, Filename)
-            elif Path is 'NULL':
-                VideoURL = "https://{}.s3.amazonaws.com/{}".format(Bucket, Filename)
+            if Path is None:
+                Path = 'null'
```
Comment (gabinante) [inline poll.py:32]: Now you're changing what's committed to dynamo and the other functions expect. so we can't merge this without breaking stuff.

Comment (gabinante) [reply poll.py:32]: FYI this was put in place because you can't put empty strings in dynamo. kill me

Comment (gabinante) [reply poll.py:32]: Also, I don't think "is None" always evaluates properly. I think I tried this before I did it the current way.

**Review by gabinante — CHANGES_REQUESTED** [review]
Either no, or you need to change what all the other functions expect when 'NULL' is logged to dynamoDB.

**Comment by jonathanjtan** [comment]
i'm silly i fixed this in my other pr

**Pull request closed without merging**